integration/class-surge.php

```diff
@@ -0,0 +1,184 @@
+<?php
+/**
+ * Surge integration file.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Integration;
+
+use function Activitypub\user_can_activitypub;
+
+/**
+ * Surge Cache integration.
+ *
+ * This class handles the compatibility with the Surge plugin.
+ *
+ * @see https://wordpress.org/plugins/surge/
+ */
+class Surge {
+	/**
+	 * The define for the Surge cache config.
+	 *
+	 * @var string
+	 */
+	public static $cache_config = 'define( \'WP_CACHE_CONFIG\', \'' . ACTIVITYPUB_PLUGIN_DIR . 'integration/surge-cache-config.php\' );';
```

Comment:
```suggestion
	public static $cache_config = "define( 'WP_CACHE_CONFIG', '' . ACTIVITYPUB_PLUGIN_DIR . 'integration/surge-cache-config.php' );";
```
Let's use double-quotes to make this easier to read. phpcs is fine with double quotes if they're used to contain single quotes.

Reply:
that was phpcbf I think or phpcs! I would love to!